src/main/java/it/gov/pagopa/cgn/portal/email/EmailNotificationFacade.java

```diff
@@ -224,11 +234,12 @@ public EmailNotificationFacade(TemplateEngine htmlTemplateEngine,
         this.configProperties = configProperties;
     }
 
-    private EmailParams createEmailParams(String mailTo, String subject, String body, String failureMessage) {
-        return createEmailParams(mailTo, Optional.empty(), subject, body, failureMessage);
+    private EmailParams createEmailParams(String mailTo, List<String> ccList, String subject, String body, String failureMessage) {
+        return createEmailParams(mailTo, ccList, Optional.empty(), subject, body, failureMessage);
     }
 
     private EmailParams createEmailParams(String mailTo,
+                                          List<String> ccList,
```

Comment:
This could have been an optional for the list of string ccList, so that the builder would have taken it if present.